_security-analytics/sec-analytics-config/detectors-config.md

```diff
@@ -28,6 +28,8 @@ You can define a new detector by naming the detector, selecting a data source an
 * Use the **Log type**, **Rule severity**, and **Source** dropdown lists to filter the rules you want to select from. 
 * Use the **Search** bar to search for specific rules.
 
+When selecting **Network events**, **Cloud Trail logs**, or **S3 access logs** as the log type, a detector dashboard is automatically created. The dashboard offers visualizations for the detector and can provide insights into data generated by the detector. For more information about visualizations, see [Building data visualizations]({{site.url}}{{site.baseurl}}/dashboards/visualize/viz-index/).
```

Comment:
Good suggestion. Thanks. To avoid using "provides" twice in the same sentence, I reworded like so:

The dashboard offers visualizations for the detector and can provide security related insights into log source data.